### PR TITLE
Add lastModified Field to Line Item Json

### PR DESCRIPTION
**Pull request by kelvin-chappell**
So that we can tell which line items are worth checking for changes in status.

### CODE REVIEW

File: common/app/common/configuration.scala [hunk 1]
```diff
@@ -360,7 +360,8 @@ class GuardianConfiguration(val application: String, val webappConfDirectory: St
         oauthCallback <- configuration.getStringProperty("faciatool.oauth.callback")
       } yield OAuthCredentials(oauthClientId, oauthSecret, oauthCallback)
 
-    val showTestContainers = configuration.getStringProperty("faciatool.show_test_containers").exists(_ == "true")
+    val showTestContainers =
+      configuration.getStringProperty("faciatool.show_test_containers").contains("true")
```
Comment (janua) [inline common/app/common/configuration.scala]: `Option.contains` isn't in scala 2.10. It's bitten me a few times though.

Might want to double check what `PROD` is on, but I think this has failed in teamcity for me before.

**Comment by janua** [comment]
:+1:

File: common/app/common/configuration.scala (hunk 1 above)
Comment (kelvin-chappell) [inline common/app/common/configuration.scala]: Thanks.  I thought it had been there forever.  I'll check this.

Comment (janua) [inline common/app/common/configuration.scala]: Yeah it's surprisingly quite new! And `idea` hints you should change it!

Comment (kelvin-chappell) [inline common/app/common/configuration.scala]: Teamcity is using 2.11 and Scala isn't used beyond there so looks safe.

**Pull request merged**